### PR TITLE
vsts: Fix release pipeline to pick local nugets

### PR DESCRIPTION
**Pull request by abhipsaMisra**

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -69,7 +69,8 @@ Param(
     [string] $verbosity = "q",
     [string] $framework = "*",
     [switch] $skipIotHubTests,
-    [switch] $skipDPSTests
+    [switch] $skipDPSTests,
+    [switch] $noBuildBeforeTesting
```
Comment (prmathur-microsoft) [inline build.ps1:73]: Why do we have skipDPSTests ?

Comment (abhipsaMisra) [reply build.ps1:73]: this is for the check where, if nothing changed in dps folder, we don't need to run the dps tests.
This is valid for gated build only, not nightly or release.

**Pull request merged**